components/productions/index/productions.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <section class="productions-container" aria-label="List of productions">
+  <section aria-label="List of productions">
```

Comment:
Nit: I'm not sure if we need the `aria-label` here by the way, we also have the `h2` right below. So screen readers would read both "List of productions" (in English!) and then "Producties" right afterward. But technically unrelated to this PR.